koku/api/report/test/tests_serializers.py

```diff
@@ -399,19 +398,3 @@ def test_invalid_delta(self):
         serializer = QueryParamSerializer(data=query_params, context={'request': req})
         with self.assertRaises(serializers.ValidationError):
             serializer.is_valid(raise_exception=True)
-
-        query_params = {'delta': 'usage'}
-        req = Mock(path='/api/cost-management/v1/reports/aws/storage/')
-        serializer = QueryParamSerializer(data=query_params, context={'request': req})
-        try:
-            serializer.is_valid(raise_exception=True)
-        except serializers.ValidationError:
-            self.fail('Serizalizer raised error for valid delta')
-
-        query_params = {'delta': 'cost'}
-        req = Mock(path='/api/cost-management/v1/reports/aws/costs/')
-        serializer = QueryParamSerializer(data=query_params, context={'request': req})
-        try:
-            serializer.is_valid(raise_exception=True)
-        except serializers.ValidationError:
-            self.fail('Serizalizer raised error for valid delta')
```

Comment:
Duplicate code. It looks like this segment was copy and pasted a few too many times.